Description: revert wrong upstream commit that causes a build failure.
 Reverts upstream commit fb964cdcd8fddd7722826379c28bcbece7b1d8fd, which
 wrongly claims to be fixing C++ compilation.  There's a better fix for that,
 stop using a C++ compiler to build your C program!
Author: Steve Langasek <vorlon@debian.org>

Index: ft2demos-2.7.1/src/ftgrid.c
===================================================================
--- ft2demos-2.7.1.orig/src/ftgrid.c	2017-03-08 18:40:55.118102280 +0100
+++ ft2demos-2.7.1/src/ftgrid.c	2017-03-08 18:40:55.114102261 +0100
@@ -54,10 +54,10 @@
   struct  AF_GlyphHintsRec_;
   typedef struct AF_GlyphHintsRec_*  AF_GlyphHints;
 
-  extern int            _af_debug_disable_horz_hints;
-  extern int            _af_debug_disable_vert_hints;
-  extern int            _af_debug_disable_blue_hints;
-  extern AF_GlyphHints  _af_debug_hints;
+  int            _af_debug_disable_horz_hints;
+  int            _af_debug_disable_vert_hints;
+  int            _af_debug_disable_blue_hints;
+  AF_GlyphHints  _af_debug_hints;
 
 #ifdef __cplusplus
   extern "C" {
